service/remote.go

```diff
@@ -127,7 +134,9 @@ func (r *RemoteService) AddRemoteBindingListener(bindingListener cluster.RemoteB
 func (r *RemoteService) Call(ctx context.Context, req *protos.Request) (*protos.Response, error) {
 	c, err := util.GetContextFromRequest(req, r.server.ID)
 	c = util.StartSpanFromRequest(c, r.server.ID, req.GetMsg().GetRoute())
-	var res *protos.Response
+	res := r.responsePool.Get().(*protos.Response)
+	res.Reset()
+	defer r.responsePool.Put(res)
```

Comment:
given that this Pool will be used by many threads, don't we have a racing condition here, where other thread gets the same "res" object that is being used by the call stack after the return of this function?

Reply:
> A Pool is safe for use by multiple goroutines simultaneously.
sync.Pool's  Get() and Put()  are both concurrently safe.

so，we don't got data race here.

Reply:
Sorry for did not reply in time!
because it is the Dragon Boat Festival holiday in my country these few days.

Reply:
no problem!